mlflow/utils/autologging_utils.py

```diff
@@ -89,3 +92,21 @@ def log_fn_args_as_params(fn, args, kwargs, unlogged=[]):  # pylint: disable=W01
     for param_name in kwargs:
         if param_name not in unlogged:
             try_mlflow_log(mlflow.log_param, param_name, kwargs[param_name])
+
+
+def wrap_patch(destination, name, original, patch, settings=None):
+    """
+    Apply a patch while preserving the attributes (e.g. __doc__) of an original function.
+
+    :param destination: Patch destination
+    :param name: Name of the attribute at the destination
+    :param original: Original function
```

Comment:
This argument feels a bit redundant given that we have `name`. Can we do something like `getattr(destination, 'name')` to infer `original`? This would simplify the usage; here's a Keras example - 

Before:

```
wrap_patch(keras.Model, "fit", keras.Model.fit, fit, settings=settings)
```

After:
```
wrap_patch(keras.Model, "fit", fit, settings=settings)
```

Reply:
makes sense. I'll make the change.